noxfiles/constants_nox.py

```diff
@@ -38,27 +39,41 @@
 # CI env variable is always set to true in Github Actions
 # The else statement is required due to the way commmands are structured and is arbitrary.
 CI_ARGS = "-T" if getenv("CI") else "--user=root"
+CI_ARGS_EXEC = "-t" if not getenv("CI") else "--user=root"
 
 # If FIDES__CLI__ANALYTICS_ID is set in the local environment, use its value as the analytics_id
 ANALYTICS_ID_OVERRIDE = ("-e", "FIDES__CLI__ANALYTICS_ID")
 ANALYTICS_OPT_OUT = ("-e", "ANALYTICS_OPT_OUT")
 
 # Reusable Commands
-RUN = (
+LOGIN = (
     "docker",
-    "compose",
-    "run",
-    "-e",
-    "VAULT_ADDR",
-    "-e",
-    "VAULT_NAMESPACE",
-    "-e",
-    "VAULT_TOKEN",
-    "--rm",
+    "exec",
+    "fides-fides-1",
+    "fides",
+    "user",
+    "login",
+    "-u",
+    "root_user",
```

Comment:
Is it possible to pull in the root user's username and password dynamically from the `fides.toml` file? Not a huge deal but this would break if that changes in the future

Reply:
Currently no. Originally I had this as a feature, but realistically 99.9% of users in a "real" environment shouldn't be using the root credentials so I didn't code that feature in.

Agreed hardcoding is suboptimal, but we've standardized on these default credentials across both core and plus. Additionally it should be pretty obvious if someone decides to change them, the failed login message would be clear I think